### DISCUSSION TITLE
Using Hacktoberfest as a tool to get updated packages

### DISCUSSION THREAD

**Discussion by pratikpc** [microsoft/vcpkg#19132] [Ideas]
While the major issue with Hacktoberfest has been the spam generated by people editing READMEs, I think a lot of Hacktoberfest-ers could be used as a resource somewhat positively to get updated packages

**Comment by jbbjarnason** [comment]
This is a great time (hacktoberfest) to either fix or add new packages from developers already using vcpkg, but haven't made time to do a PR to the official repo.

**Comment by braindigitalis** [comment]
A year later and bumping this. i made a PR this year that fell within hacktoberfest without even realising it. It would be nice if PRs for updates to packages were accepted as part of hacktoberfest, but i can understand why they wouldnt want the hassle of people making pointless vanity edits and raising phoney PRs.
